[rospy.client][INFO] 2025-06-24 20:52:41,993: init_node, name[/urdf_spawner], pid[12394]
[xmlrpc][INFO] 2025-06-24 20:52:41,993: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2025-06-24 20:52:41,993: Started XML-RPC server [http://192.168.1.124:33325/]
[rospy.impl.masterslave][INFO] 2025-06-24 20:52:41,993: _ready: http://192.168.1.124:33325/
[rospy.init][INFO] 2025-06-24 20:52:41,994: ROS Slave URI: [http://192.168.1.124:33325/]
[xmlrpc][INFO] 2025-06-24 20:52:41,994: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2025-06-24 20:52:41,994: Registering with master node http://localhost:11311
[rospy.init][INFO] 2025-06-24 20:52:42,094: registered with master
[rospy.rosout][INFO] 2025-06-24 20:52:42,094: initializing /rosout core topic
[rospy.rosout][INFO] 2025-06-24 20:52:42,101: connected to core topic /rosout
[rospy.simtime][INFO] 2025-06-24 20:52:42,104: initializing /clock core topic
[rospy.simtime][INFO] 2025-06-24 20:52:42,111: connected to core topic /clock
[rosout][INFO] 2025-06-24 20:52:42,122: Loading model XML from ros parameter
[rosout][INFO] 2025-06-24 20:52:42,125: Waiting for service /gazebo/spawn_urdf_model
[rosout][INFO] 2025-06-24 20:52:42,129: Calling service /gazebo/spawn_urdf_model
[rosout][INFO] 2025-06-24 20:52:42,153: Spawn status: SpawnModel: Entity pushed to spawn queue, but spawn service timed out waiting for entity to appear in simulation under the name my_robot
[rospy.internal][INFO] 2025-06-24 20:52:42,159: topic[/clock] adding connection to [http://192.168.1.124:42777/], count 0
[rospy.core][INFO] 2025-06-24 20:52:42,228: signal_shutdown [atexit]
[rospy.internal][INFO] 2025-06-24 20:52:42,230: topic[/clock] removing connection to http://192.168.1.124:42777/
[rospy.impl.masterslave][INFO] 2025-06-24 20:52:42,230: atexit
[rospy.internal][WARNING] 2025-06-24 20:52:42,231: Unknown error initiating TCP/IP socket to 192.168.1.124:60095 (http://192.168.1.124:42777/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 651, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

